examples/perfect_random_model_example.py

```diff
@@ -0,0 +1,66 @@
+# ## Illustrate usage of DAPPER to (interactively) run a synthetic ("twin") experiment.
+
+# #### Imports
+# <b>NB:</b> If you're on <mark><b>Gooble Colab</b></mark>,
+# then replace `%matplotlib notebook` below by
+# `!python -m pip install git+https://github.com/nansencenter/DAPPER.git` .
+# Also note that liveplotting does not work on Colab.
+
+# %matplotlib notebook
+from mpl_tools import is_notebook_or_qt as nb
+
+import dapper as dpr
+import dapper.da_methods as da
+from dapper.mods.L96s.Grudzien2020 import (HMM_em_ensemble, HMM_rk_ensemble,
+                                           HMM_truth)
+
+# #### Load experiment setup: the hidden Markov model (HMM)
+
+
+# #### Generate the same random numbers each time this script is run
+
+seed = dpr.set_seed(3000)
+
+# #### Simulate synthetic truth (xx) and noisy obs (yy)
+xx, yy = HMM_truth.simulate()
+
+# note - the truth twin takes twice the number of the integration steps
+# as the model twin in this configuration
+xx = xx[0::2, :]
```

Comment:
This is the solution I would have suggested too :-)

One minor suggestion:

```suggestion
xx = xx[::2]
```

Reply:
Cool, I don't remember the Python syntax that well, if you say that is the correct slicing let's do it.